src/migrator/utils.ts

```diff
@@ -425,7 +439,8 @@ LIMIT 1;`,
 			migrated_row_count: data['migrated row count'],
 			error_count: data['error count'],
 			converged_time: data['converged time'],
-			is_backing_off: data['is backing off'] === 1,
+			is_backing_off:
+				data['is backing off'] === true || data['is backing off'] === 1,
```

Comment:
Afaict that's just for being able to do this major bump right?
Should this and the above `typeof data.executed_migrations === 'string'` have a `TODO` to drop it on a next major?

Reply:
It's also necessary for sqlite/websql where BOOLEAN is just an alias for INTEGER